backend/src/db/migrations/20230821120000_create_user_details_table.ts

```diff
@@ -0,0 +1,41 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'user_details';
+
+const ColumnName = {
+  FIRST_NAME: 'first_name',
+  LAST_NAME: 'last_name',
+  USER_ID: 'user_id',
+  CREATED_AT: 'created_at',
+  UPDATED_AT: 'updated_at',
+};
+
+const up = (knex: Knex): Promise<void> => {
+  return knex.schema.createTable(TABLE_NAME, (table) => {
+    table.increments();
```

Comment:
Define `id` col explicitly
```suggestion
   table.increments(ColumnName.ID).primary();
```

Reply:
Comments should be resolved only in case if they were fixed

Reply:
Okay, got it. I've fixed everything but resolved the comments before pushing.